### DISCUSSION TITLE
`group-[.map-block--collapsed]:relative` isn't generating CSS

### DISCUSSION THREAD

**Discussion by janbaykara** [tailwindlabs/tailwindcss#9864] [Help]
Hi there. I'm making use of [arbitrary group syntax](https://tailwindcss.com/docs/hover-focus-and-other-states#arbitrary-groups) like `group-[.map-block--collapsed]:relative`, but finding that the CSS isn't being generated. Am I missing a trick?

Tailwind's VSCode plugin correctly recognises the syntax, but the JIT compiler just isn't pushing it out despite it being there in the HTML (and in the path of `content`):

<img width="623" alt="Screenshot 2022-11-17 at 14 56 16" src="https://user-images.githubusercontent.com/237556/202480152-0d9ec5d1-b28d-470f-906e-db08c4d0b3a9.png">

**Comment by wongjn** [comment]
Seems to be generate fine in the Tailwind Play sandbox, so it suggests there could be some misconfiguration within the project. Perhaps check:
- Tailwind v3.2 or higher is installed.
- The file the code is in is covered by the `content` globs.

Otherwise, consider providing a remotely-hosted git repository that we can take a closer look at.
